### ISSUE TITLE
Topiikkia asettavat / indikoivat sanat ja tämän konseptin määrittely

### ISSUE THREAD

**Issue by hrmJ** [hrmJ/100-tapaa-indikoida-topiikkia#18]
@Kirde80 Pitäisikö teoriaosassa määritellä jotenkin se, mitä on topiikin asettaminen / indikoiminen? (tai siis tietysti pitäisi, mutta...)  Itse asiassa mietin, onko kyseessä ehkä sittenkin kaksi eri asiaa.  Esimerkiksi tässä kappaleessa 

> Aloin etsimään **asuntoa** heti harjoittelupaikan varmistuttua, joka ei Hollannissa niin helppoa 
> ulkomaalaiselle ole. Tähän nähden löysin asunnon suhteellisen vaivatta, mutta kannattaa kirjautua  asunnonhaku-sivustolle ja maksaa pieni maksu tästä hyvästä, sillä muuten hakeminen 
> vuokranvälitysfirmojen tms. kautta tulee kuitenkin....

voisi ehkä sanoa, että topiikki *asetetaan* ensimmäisessä lauseessa ja topiikkia *indikoi* sana asunto.
Toisin sanoen, voisiko sen indikoimisen määrittää aika spesifisti jotenkin sillä tavalla, että kyseessä on *se leksikaalinen vihje, jonka perusteella kappaleelle on tulkittavissa tietty topiikki*? 

Rupesin miettimään tätä kirjoittaessani [tätä riviä](https://github.com/hrmJ/100-tapaa-indikoida-topiikkia/blame/master/metodista.Rmd#L63)  ja pohdin, pitäisikö ajatus topiikin paikantamisesta yhdellä sanalla selostaa kokonaan metodin puolella vai osittain jossain muualla

**Comment by Kirde80** [comment]
@hrmJ Joo olet oikeassa, kyllä me tarvitaan tuo erottelu. Ja sun spesifi määritelmäkin näyttää äkkiseltään hyvältä. Ehkä ratkaisu voisi olla se, että teoriaosassa puhuttaisiin topiikin asettamisesta, koska sitä käytetään meidän lähdekirjallisuudessakin, ja sitten metodin puolella voisi olla tämä meidän tarkennus leksikaaliseen vihjeeseen ja topiikin paikantamiseen yhdellä sanalla?

**Comment by hrmJ** [comment]
Joo, hyvä. Mä koitan mietttiä jotain sopivaa muotoilua metodiosaan

**Comment by Kirde80** [comment]
Huomenta! Olin eilisen työkoneen varassa ja nyt taas omalla masiinalla.
Olen ehkä hiukan konfuusiossa, että paikallistan nämä muutokset GitHubista,
mutta yritän! Saatan siis kohta kysellä, jos en löydä :D

2018-01-26 7:46 GMT+02:00 Juho Härme <notifications@github.com>:

**Comment by hrmJ** [comment]
Joo siis tää viesti minkä sä oot näämmä saanu ei ollut mikään oleellinen, mä vaan "suljin" kysymyksiä, jotka me oltiin mun mielestä jo ratkaistu
